test/test.ts

```diff
@@ -1,11 +1,11 @@
 import * as Token from 'token-types';
 import { assert } from 'chai';
 import * as strtok3 from '../lib';
-import * as Path from 'path';
+import * as Path from 'node:path';
```

Comment:
```suggestion
import Path from 'node:path';
```

You can default import many of the Node.js core modules.